blueprints/ember-cli-notifications/index.js

```diff
@@ -7,9 +7,11 @@ module.exports = {
   // Use config to determine whether Font Awesome is imported into consuming app
   afterInstall: function(app) {
     var projectConfig = this.project.config(app.env);
-    var config = projectConfig['ember-cli-notifications'];
+    var config = projectConfig['ember-cli-notifications'] ||
+                 app.options['ember-cli-notifications'] ||
+                 { includeFontAwesome: false };
```

Comment:
I'd actually prefer if the default value was `true`. That way the add-on will work as designed out the box with no configuration.